ASM/training-mode/Onscreen Display/Act OoJump/Aerial Out of DoubleJump.asm

```diff
@@ -0,0 +1,71 @@
+    # To be inserted at 800cc570
+    # (.fn ftCo_JumpAerial_IASA in asm/melee/ft/ft_0C88.s)
+    .include "../../Globals.s"
+
+    .set playerdata, 30
+    .set player, 29
+
+##########################################################
+## 804a1f5c -> 804a1fd4 = Static Stock Icon Text Struct ##
+## Is 0x80 long and is zero'd at the start ##
+## of every VS Match ##
+## Store Text Info here ##
+##########################################################
```

Comment:
I'm pretty sure this comment is outdated and incorrect. Can you remove it from each of the asm files.